src/content/en/updates/posts/2013/12/300ms-tap-delay-gone-away.markdown

```diff
@@ -12,105 +12,46 @@ tags:
-
-<h2>How Chrome removed the 300ms delay</h2>
+Ever since the first release of Chrome for Android, this delay was removed if pinch-zoom was also disabled. However, pinch zoom is an important accessibility feature. As of Chrome 32 (back in 2014) this **delay is gone** for **mobile-optimised** sites, **without removing pinch-zooming**! Firefox and IE/Edge did the same shortly afterwards, and in March 2016 a similar fix landed in iOS 9.3.
 
-<p>Chrome and Firefox for Android have, for some time now, removed the 300ms tap delay for pages with this:</p>
+The performance difference is huge:
 
-{% highlight HTML %}
-<meta name="viewport" content="width=device-width, user-scalable=no">
-{% endhighlight %}
-
-<p>Pages with this cannot be zoomed, therefore "double-tap to zoom" isn't an interaction, therefore there's no need to wait for double-taps. However, we also lose pinch-zooming.</p>
+{% ytvideo AjUpiwvIa5A?rel=0 %}
 
-<p>Pinch-zooming is great for taking a closer look at a photo, some small print, or dealing with a set buttons/links that are placed too closely together. It's an out-of-the-box accessibility feature.</p>
+Having a UI that responds instantly means the user can quickly press each button with confidence, rather than pausing and waiting for a response. Find out more about the impact of human reaction times and web performance in our [introduction to RAIL](/web/tools/chrome-devtools/profile/evaluate-performance/rail).
 
-<p>If a site has…</p>
+To remove the 300-350ms tap delay, all you need is the following in the `<head>` of your page:
 
 {% highlight HTML %}
 <meta name="viewport" content="width=device-width">
 {% endhighlight %}
 
-<p>…double-tap zooms in a little bit. Not a particularly useful amount. A further double-tap zooms back out. We feel this feature, on mobile-optimised pages, isn't useful. So we removed it! This means we can treat taps as clicks instantly, but we <strong>retain pinch-zooming</strong>.</p>
-
-<h2>Is this change an accessibility concern?</h2>
-
-<p>We don't believe so, but the reason we release beta versions of Chrome is so users can try new features and give us feedback.</p>
-
-<p>We tried to imagine a user this may affect, someone who:</p>
-
-<ul>
-<li>has a motor impairment that prevents multi-touch interaction such as pinch-zoom, but not two taps in the same area within 300ms</li>
-<li>has a minor visual impairment that is overcome by the small amount of zooming provided by double-tap on mobile optimised sites</li>
-</ul>
-
-<p>But they're catered for by the text sizing tools in Chrome's settings, or the screen magnifier in Android, which covers all sites and native apps, and can be activated by triple-tap.</p>
+This sets the viewport width to the same as the device, and is general best-practice for mobile-optimised sites. With this tag, browsers assume you've made text readable on mobile, and the double-tap-to-zoom feature is dropped in favour of faster clicks.
```

Comment:
'and is general best-practice' > 'and is generally a best practice'. Please fix.